src/components/screens/dashboard/dashboard-screen.tsx

```diff
@@ -1,18 +1,62 @@
-import { useAtom } from "jotai";
-import { authAtom } from "../../../atoms/auth";
+import { useAtomValue } from "jotai";
+import { authAtom, userProfileAtom } from "../../../atoms/auth";
+import { useEffect, useState } from "react";
+import { useApi } from "../../../hooks/use-api";
+import { PersonTotalTime } from "../../../generated/client";
 
 /**
  * Dashboard screen component
  * 
  */
-const DashboardScreen = () => {
-  const [auth] = useAtom(authAtom);
+function DashboardScreen () {
+  const auth = useAtomValue(authAtom)
+  const userProfile = useAtomValue(userProfileAtom);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [errorMsg, setErrorMsg] = useState<string>();
+  const [personTotalTime, setPersonTotalTime] = useState<PersonTotalTime[]>();
+  const { personsApi } = useApi();
+  
+/**
+ * Initialize logged in person's time data.
+ * 
+ */
+  const getPersons = async () => {
+    setIsLoading(true);
+    //NOTE: Trainees do not have time bank entries, therefore a use keycloak ID from another employee for testing.
+    const loggedInPerson = await (await personsApi.listPersons({ active: true })).filter(person => person.keycloakId === userProfile?.id)
+
+    if (loggedInPerson.length > 0) {
+      try {
+        if (auth?.tokenRaw){
+          const fetchedPerson : PersonTotalTime[] = await personsApi.listPersonTotalTime({personId: loggedInPerson[0].id});
+          setPersonTotalTime(fetchedPerson);
+        }
+      } catch (error) {
+        console.error(error);
```

Comment:
We have an error handler to use now :), should set all the errors to errorAtom.
For example:
`const [ error, setError ] = useAtom(errorAtom); // This line is for upper scope`
`setError(`${ "Person fetch has failed." }, ${ error }`);`

Reply:
Fixed